frontend/src/app/components/work-packages/wp-relations-count/wp-relations-count.component.ts

```diff
@@ -1,8 +1,8 @@
 import {Component, Input, OnInit} from '@angular/core';
 import {WorkPackageRelationsService} from '../../wp-relations/wp-relations.service';
-import {WorkPackageCacheService} from '../../work-packages/work-package-cache.service';
 import {combineLatest} from 'rxjs';
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e71869ec897476f98b23dceb4c3c60722bb3632a/frontend/src/app/components/work-packages/wp-relations-count/wp-relations-count.component.ts#L6"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>